teleport.lua

```diff
@@ -29,23 +29,7 @@ Examples:
 ]====]
 
 function teleport(unit,pos)
```

Comment:
Looks like there are three scripts that import and call this script's `teleport()`: `deep-embark`, `flashstep`, and `gui/teleport`. Should we migrate those as well? I'm fine putting those in another PR if you prefer.

Reply:
you mean have them call `dfhack.units.teleport()` directly instead of bouncing off `teleport.teleport()`? Yeah I could totally do that here. One sec.

Reply:
Note that I'm still leaving the `teleport.teleport` function exported since out of tree scripts may still be calling it.

Reply:
done

Reply:
Yep, 97da3a82adb45025a9d2cac40bbb143671befd99 is exactly what I was thinking of. Thanks!